src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -20,6 +20,7 @@ azure-loganalytics==0.1.0
 azure-mgmt-advisor==9.0.0
 azure-mgmt-apimanagement==3.0.0
 azure-mgmt-appconfiguration==2.2.0
+azure-mgmt-appcontainers==1.0.0
```

Comment:
Just out of curiosity, may I ask why SDK package `azure-mgmt-appcontainers` already has version `2.0.0`, but you still bump the version `1.0.0`?